### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by blasten**
At some point, tedium will auto generate the readme files. https://github.com/PolymerLabs/tedium/issues/17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,177 +2,33 @@
...
-It's a set of experimental layout elements for your app. It includes:
+A set of experimental layout elements for your app. It includes:
 
-- [app-toolbar](#app-toolbar)
-- [app-header](#app-header)
-- [app-header-layout](#app-header-layout)
-- [app-drawer](#app-drawer)
-- [app-drawer-layout](#app-drawer-layout)
-- [app-scrim](#app-scrim)
+- [app-toolbar](/app-toolbar) - A horizontal bar containing items that can be used for label, navigation, search and actions.
```
Comment (keanulee) [inline README.md]: `A horizontal toolbar`

File: README.md [hunk 2]
```diff
@@ -2,177 +2,33 @@
...
----------
+- [app-header](/app-header) - A container for app-toolbars that can have scroll effects.
 
-app-header-layout is a layout element that contains a header and a content.
-This layout element uses the document scroll by default, but this element can 
-also define its own scrolling region.
+- [app-header-layout](/app-header-layout) - A wrapper that lays out an app-header and the content.
 
-#### Example
+- [app-drawer](/app-drawer) - A navigation drawers that can slide in from the left, right, bottom or top.
```
Comment (keanulee) [inline README.md]: drawers -> drawer

File: app-drawer-layout/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+##&lt;app-drawer-layout&gt;
+
+app-drawer-layout is a layout element that contains a drawer and a main content.
+When the viewport width is smaller than `responsiveWidth`, this element changes
+to narrow layout. In narrow layout, the drawer will be stacked on top of the main content.
+The drawer will slide in/out to hide/reveal the main content.
+
+
+By default the drawer is in the left side of the screen:
```
Comment (keanulee) [inline app-drawer-layout/README.md]: `is on the left side`

File: app-drawer-layout/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  <div drawer>
+    <!-- drawer content -->
+  </div>
+  <div main>
+    <!-- main content -->
+  </div>
+</app-drawer-layout>
+```
+
+A drawer in right side of the screen:
```
Comment (keanulee) [inline app-drawer-layout/README.md]: `on the right side`

**Comment by frankiefu** [comment]
Thanks all.  LGTM.

**Pull request merged**